.github/workflows/main.yml

```diff
@@ -1,4 +1,4 @@
-name: main-commit-checks
+name: cypress-tests
```

Comment:
Can we match yaml file name of with workflow name? Maybe `main-tests`? As it's doing lint and format check also.